code/modules/client/preference_setup/loadout/loadout.dm

```diff
@@ -60,16 +60,14 @@ var/list/gear_datums = list()
 	var/mob/preference_mob = preference_mob()
 	for(var/gear_name in gear_datums)
 		var/datum/gear/G = gear_datums[gear_name]
+		if(max_cost && G.cost > max_cost)
+			continue
 		if(G.whitelisted && preference_mob)
 			for(var/species in G.whitelisted)
 				if(is_alien_whitelisted(preference_mob, species))
 					. += gear_name
 					break
 
```

Comment:
Add an
`else
  . += gear_name`
In here otherwise it will not allow items that aren't whitelisted to get through